uproot/requirements.txt

```
@@ -1,5 +1,5 @@
-uproot==5.1.2
-awkward==2.5.0
-dask-awkward==2023.11.3
-vector==1.1.1.post1
-pyarrow==14.0.1
+uproot==5.3.8
```

```suggestion
uproot==5.3.9
```

`uproot` `v5.3.9` _just_ came out today. https://github.com/scikit-hep/uproot5/releases/tag/v5.3.9